Code/Python/check pangram.py

```diff
@@ -0,0 +1,46 @@
+#Program to check if a given string is pangram or not
+
+#A pangram is a sentence containing every letter in the English Alphabet.
+
+
+#importing the string module
+from string import *
+
+#Function to check the string
+def pangram(string):
+    
+    
+    for alpha in ascii_lowercase:
+        
+        #checking for all the alphabets in the input string
+        #converting all the letters in lowercase
+        
+        if alpha not in string.lower():   
+            return False
+    
+    
+    return True
+
+
+#main
+string=input("Enter any string: ")
```

Comment:
```suggestion
string = input("Enter any string: ")
```
Add space around the operators like <, > . =. == etc

Reply:
done!